### PR TITLE
Templates fix

### PR DESCRIPTION
**Pull request by anton-bondarev**
Fix build for following templates:

- arm/stm32_vl_light
- arm/omap
- arm/stm32_dvfs
- arm/stm32_modbus

### CODE REVIEW

File: templates/arm/ti816x-qt/build.conf [hunk 1]
```diff
@@ -16,6 +16,8 @@ CXXFLAGS += -march=armv7-a -mtune=cortex-a8 -mno-unaligned-access
 CXXFLAGS += -fno-rtti
 CXXFLAGS += -fno-exceptions
 CXXFLAGS += -fno-threadsafe-statics
+ /* C++14 */
+CXXFLAGS += -fsized-deallocation
```
Comment (AntonKozlov) [inline templates/arm/ti816x-qt/build.conf]:
```
Enable the built-in global declarations
                   void operator delete (void *, std::size_t) noexcept;
                   void operator delete[] (void *, std::size_t) noexcept;
```
Intention is not clear. Could you describe for what is this?

Comment (anton-bondarev) [reply templates/arm/ti816x-qt/build.conf]: without this flag I got error
```/home/anton/embox/src/compat/cxx/include/standalone_new:49:6: error: 'void operator delete(void*, size_t)' is a usual (non-placement) deallocation function in C++14 (or with -fsized-deallocation) [-Werror=c++14-compat]```

Comment (AntonKozlov) [reply templates/arm/ti816x-qt/build.conf]: Description of the warning is https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wc_002b_002b14-compat
seems like it should catch things that will break when the c++ change language version, i.e. switch from c++11 to c++14.
c++ code also seems to built with -Wall -Werror, compiler have more warnings than expected. Embox should not insist on third-party to compile with those options. If third-party project request them, it's not right for them, suppose there is something like `--disable-warning-as-erros` configure switch.
Does `include/standalone_new` lay in include directory passed as `-isystem`? System includes should not trigger all warnings.
It's also possible to disable the `c++14-compat` warning for the third-party or, at last, globally

Comment (anton-bondarev) [reply templates/arm/ti816x-qt/build.conf]: Yes, It's a new for c++14.
The warning appears during src/compat/cxx/core/new.cpp compilation not third-party
We can disable c++14-compat globaly, but I don't understand why -fsized-deallocation is bad?

Comment (anton-bondarev) [reply templates/arm/ti816x-qt/build.conf]: @AntonKozlov I move commits with c++ to another branch

**Pull request merged**